src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let b = 0;
+
+  const infinit = (a) => {
+    if (a || a === 0) {
+      b = b + a;
+
+      return infinit;
+    }
+
+    const ba = b;
```

Comment:
it's just concatenated variable names? :) Let's replace a, b, ba with meaningful variable names
Even in this small function, it's easy to misunderstand something with such variables